doc/security/specs/secure_boot/index.md

```diff
@@ -43,17 +42,15 @@ firmware upgrade support.
 The boot process flows as follows:
 
 1.  Power on.
-    1.  Execution is restricted to the ROM region via
-        [OpenTitan Secure Boot HW Support][secure-boot-hw-support].
-    2.  The last PMP region (region #15) is configured by reset logic to cover
-        the entire flash region, with only the R and L bits set (that is, the
-        region is **R**ead-only and **L**ocked, meaning the region is respected
-        in Machine mode and cannot be unlocked unless by a system reset).
+    1.  Execution is restricted to the ROM region. Execution of other types of
+        memory is initially prevented by the reset logic.
```

Comment:
I've intentionally left this vague for the time being. I'm not certain whether reset logic will configure ePMP a certain way or if we will just rely on default noexec behavior for memories other than the ROM.